README.md

```diff
@@ -91,7 +91,10 @@ level = "info"
 [data]
 # Folder for buffering received data
 path = "/data/carbon-clickhouse/"
-# Rotate (and upload) file interval.
+# Rotate (and upload) file iniciated on size and interval
+# Rotate (and upload) file size
```

Comment:
It looks like bytes to me. Would be good to add units here

Reply:
It's a byte, yes. It's possible add unit (k, m, g).